### PR TITLE
Simple

### PR DESCRIPTION
**Pull request by mre**

### CODE REVIEW

File: content/2024/be-simple/index.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+## Simple Means Letting Go
+
+Perhaps another way to explain it is the midwit theme:
+
+{{ figure(src="midwit.jpg") }}
+
+I try to stay on the left side of this curve as much as I can.
+
+That is, I'm aware that there are "smarter" ways to do things, but I don't want to dedicate time to learning about them.
+Knowing that getting over the middle-ground takes lots of effort keeps me on the left side.
```
Comment (M3t0r) [inline content/2024/be-simple/index.md]: It's wrong, but I don't even know what you're trying to say :-/

File: content/2024/be-simple/index.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+- My shoes are 6 years old. So is my wardrobe. 
+- No meetings if possible. I'm the guy who sits at his desk for 8 hours straight, only getting up for bathroom breaks twice.
+- No property; I'm a happy tenant.
+- No commute; I work remotely.
+
+Great home cinema setup you have there.
+Thanks for inviting me over!
+At home, I don't even have external speakers.
+
+You're planning a trip to the Bahamas? Enjoy! Send me a photo, maybe.
```
Comment (M3t0r) [inline content/2024/be-simple/index.md]:
```suggestion
You're planning a trip to the Bahamas? Enjoy! Send me a photo.
```

"maybe" comes off as a bit passive aggressive. Just drop it.

**Review by M3t0r — COMMENTED** [review]
2 things, but pretty good otherwise :+1:

File: content/2024/be-simple/index.md (hunk 1 above)
Comment (mre) [reply content/2024/be-simple/index.md]: changed. what do you think?

File: content/2024/be-simple/index.md [hunk 3]
```diff
@@ -98,11 +98,10 @@ Perhaps another way to explain it is the midwit theme:
 
 I try to stay on the left side of this curve as much as I can.
 
-That is, I'm aware that there are "smarter" ways to do things, but I don't want to dedicate time to learning about them.
-Knowing that getting over the middle-ground takes lots of effort keeps me on the left side. 
-The other end of the spectrum only provides diminishing returns anyway.
-I think I made it over that hump only a few times in my life.
-For the rest, knowing that there's an awkward middle-ground makes me keep things simple.
+I'm aware that there are "smarter" ways to do things, but I don't want to dedicate time to learning about them.
```
Comment (M3t0r) [inline content/2024/be-simple/index.md:101]: Phrasing makes it seem like you don't want to learn / improve. But knowing you, you only want to spend your time on things that are important to you, not everything. So picking the things you go deep in. Might need some rephrasing still.

Comment (mre) [reply content/2024/be-simple/index.md:101]: Good point.

Comment (mre) [reply content/2024/be-simple/index.md:101]: updated

**Review by M3t0r — APPROVED** [review]

**Pull request merged**